### PR TITLE
Add organisation info

### LINKED ISSUE #1: Organisation README
- Add instructions for creating virtual environment to the readme
- Suggest users have a single 'cruise' virtual environment that they activate for all repositories
- Add advice about preparing in advance of the cruise

### LINKED ISSUE #3: Detailed setup instructions
Add a document covering:

- [x] create `go-ship/`
- [x] create `gs-venv/`
- [x] create `data/`
- [x] clone repos
- [x] Schematic?

### PR DESCRIPTION
**Pull request by jatkinson1000**
Closes #1 with `profile/README.md`
Closes #3 with `setup.md`

Added background information, simple setup (file structure, venv, data etc.), and contribution guidelines to organisation README.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+
+The software provided here is community-developed, free, and open-source.
+It has been collated under the [GO-SHIP](https://www.go-ship.org/) program, but is
+open to anyone to download and use.
+
+
+## Usage
+
+Each [repository](https://github.com/orgs/GO-SHIP-Oceanography/repositories) under
+this organisation represents a single tool for use on cruise.
```
Comment (lcimoli) [inline profile/README.md]: I would add a sentence to point out that we will use different coding languages, something like: "Please note that different repositories contain software in different programming languages, such as Python, MatLab or spreadsheets. Whenever possible (and reasonable), the code is provided in all available programming languages."

We also need to decide what to do with codes for the same purpose but written in different languages: do we keep them all in the same repository or not? What's the best practice?

File: setup.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+For example, suppose you want the
+[station_bathymetry](https://github.com/GO-SHIP-Oceanography/station_bathymetry) tool.
+following the instructions on the repository we would
+[clone it](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+using:
+```bash
+git clone https://github.com/GO-SHIP-Oceanography/station_bathymetry.git
+```
+which will create the directory `station_bathymetry/` with the code inside.  
+we can navigate to that directory using:
```
Comment (lcimoli) [inline setup.md]: We (capital W)

**Review by lcimoli — COMMENTED** [review]
Thank you so much, Jack - this looks great!
I think one thing worth mentioning is that the code will be in different programming languages (and we need to decide how to structure the repositories when we have the same code in multiple languages). Other than that, I think it reads well and all the important information is there. Thank you!

File: setup.md (hunk 2 above)
Comment (jatkinson1000) [reply setup.md]: Done

File: profile/README.md (hunk 1 above)
Comment (jatkinson1000) [reply profile/README.md]: I have added a note as you describe under `### Repositories`.

As for how to treat multiple languages, if the code is a short script and both implementations fall under the same license (and authors?) I would probably put them in the same place. This way it is also easier to try and keep updates applying to both languages, not just one.

If it is a bigger code with several files to manage, however, then we might create 2 separate repos and users can choose the appropriate one.

I'd say probably discuss with authors and try to be sensible on a case-by-case basis.

**Comment by jatkinson1000** [comment]
Hi Laura, if you are happy could you 'approve' this and merge it?
That will have things ready for meeting later today, and I will create a new branch for adding contribution guidelines (#2).

**Review by lcimoli — APPROVED** [review]
Thanks!

**Pull request merged**